2022Game/data/2022Game_label_map.pbtxt

```diff
@@ -0,0 +1,128 @@
+item {
+	id: 1
+	name: 'blue_cargo'
+}
+
+item {
+	id: 2
+	name: 'red_cargo'
+}
+
+item {
+	id: 3
+	name: 'blue_launchpad'
+}
+
+item {
+	id: 4
+	name: 'red_launchpad'
+}
+
+item {
+	id:5, 
+	name: 'rapid_react_hub_logo
+}
+
+item {
```

Comment:
These should probably be a different name than the 2020 blue/red tape corners, it'll make it easier to filter the old corners out of the 2020data.